static/src/stylesheets/module/nav/_brand-bar.scss

```diff
@@ -98,6 +100,22 @@
     }
 }
 
+.brand-bar__item__badge {
+    // TODO: Palette?
+    background: lighten(colour(guardian-brand), 43%);
```

Comment:
Should use `guss-colour(news-main-2, $pasteup-palette)` or `colour(news-accent)`